src/.vuepress/config.js

```diff
@@ -184,6 +185,10 @@ module.exports = {
           title: 'Command line interface',
           path: '/docs/06-cli/',
         },
+        {
+          title: 'Operation Guide',
+          path: '/docs/08-operation-guide/',
```

Comment:
maybe rename folder to `07-operation-guide` and rename `08-about`. This helps preserve order in the folder structure with what is shown in the navigation. I think you will only need to change the reference in this config file.

Reply:
The navigation is needs a little tweaking to work correctly. You will need to add this:
```
children: [
   '08-operation-guide/',
   '08-operation-guide/01-setup',
],
```